package.json

```diff
@@ -155,7 +155,7 @@
     "jest-environment-jsdom": "29.6.4",
     "js-yaml": "4.1.0",
     "mammoth": "1.6.0",
-    "marked": "4.3.0",
+    "marked": "9.0.0",
     "marked-terminal": "5.2.0",
```

Comment:
The 'marked' package version has been upgraded from 4.3.0 to 9.0.0. Please ensure this large version jump doesn't introduce breaking changes or incompatibility issues with the existing codebase.